src/distributions/normal.jl

```diff
@@ -490,7 +490,7 @@ end
 # precludes the usage of logPdf functions previously defined. Below function is
 # meant to be used with Zygote.
 function Distributions.logpdf(η::UnivariateNormalNaturalParameters, x)
-    return log(invsqrt2π) + x * η.weighted_mean + x^2 * η.minus_half_precision + lognormalizer(η)
+    return log(invsqrt2π) + x * η.weighted_mean + x^2 * η.minus_half_precision - lognormalizer(η)
```

Comment:
Maybe replace `log(invsqrt2π)` with ` -log2π / 2` from `SpecialFunctions.jl`

Reply:
Thanks for the nice catch! [Done](https://github.com/biaslab/ReactiveMP.jl/pull/239/commits/24c3808abbb5175dc073449dc9b26f9205f200d5)